src/aap_eda/settings/default.py

```diff
@@ -241,7 +241,7 @@ def _get_databases_settings() -> dict:
             "PASSWORD": settings.get("DB_PASSWORD"),
             "NAME": settings.get("DB_NAME", "eda"),
             "OPTIONS": {
-                "sslmode": settings.get("PGSSLMODE", default="prefer"),
+                "sslmode": settings.get("PGSSLMODE", default="allow"),
```

Comment:
Why this change?

Reply:
https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION